ruby/syntax/lexical/Whitespace.sdf3

```diff
@@ -0,0 +1,37 @@
+module Whitespace
+
+lexical syntax
+
+  // Layout whitespace
+  LAYOUT = [\ \t\n\r]
+  
+  // Block comment whitespace
```

Comment:
Why are _comments_ included in the `Whitespace` file? If we want to get a clean structure if would perhaps be more logical to put this into its own file.

Reply:
Since comments are also regarded as whitespace. They are not part of the AST. But sure, if you think that helps we can put it in a separate file.